casepro/backend/__init__.py

```diff
@@ -199,6 +199,15 @@ def fetch_contact_messages(self, org, contact, created_after, created_before):
                  Message.as_json() for incoming messages and Outgoing.as_json() for outgoing messages.
         """
 
+    @abstractmethod
+    def get_url_patterns(self):
```

Comment:
I don't think we ever agreed on a "PR from Epic" approach. @ewheeler started [a discussion](https://groups.google.com/forum/#!topic/casepro-dev/fMDUsUbco-I) about our development workflow a while back and we seemed to be agreeing on the opposite - small discrete feature PRs.

Seems like that approach is either for both of us, no?

Also am still curious what these backend URLs are going to be used for.